orchagent/vnetorch.cpp

```diff
@@ -1868,11 +1900,20 @@ void VNetRouteOrch::createBfdSession(const string& vnet, const NextHopKey& endpo
         auto tun_name = vnet_orch_->getTunnelName(vnet);
         VxlanTunnelOrch* vxlan_orch = gDirectory.get<VxlanTunnelOrch*>();
         auto tunnel_obj = vxlan_orch->getVxlanTunnel(tun_name);
+        /*
+            Even for localenpoint, we will use tunnel source IP as local_addr of BFD session.
+        */
         IpAddress src_ip = tunnel_obj->getSrcIP();
 
         FieldValueTuple fvTuple("local_addr", src_ip.to_string());
         data.push_back(fvTuple);
-        data.emplace_back("multihop", "true");
+        if (isLocalEndpoint(vnet, endpoint_addr))
+        {
+            data.emplace_back("multihop", "false");
```

Comment:
Please revert this as currently vnet only supports multihop bfd

Reply:
Addressed.